apps/site/src/lib/components/forms/shared/BaseForm.tsx

```diff
@@ -102,10 +102,17 @@ export default function BaseForm({
 		<form
 			method="post"
 			className="bg-black border-[5px] border-white text-[var(--color-white)] w-8/12 flex flex-col items-center py-12 gap-14 z-1 max-[800px]:w-9/12 max-[400px]:w-11/12 drop-shadow-[25px_33px_0px_rgba(255,255,255,1)]"
-			action="/api/user/apply"
+			action={applyPath}
 			encType="multipart/form-data"
 			onSubmit={handleSubmit}
 		>
+			<input
+				type="text"
+				name="application_type"
```

Comment:
Chore: now that we have the hidden input field, we no longer need to manually add the value in `handleSubmit`.